### PR TITLE
locations, services improvements

### PR DESCRIPTION
**Pull request by vbud**

### CODE REVIEW

File: src/constants/properties.js [hunk 1]
```diff
@@ -18,16 +18,19 @@ export const faithBased = {
...
   label: 'To get connected',
 }
 
 export const hours = {
   path: 'json.hours',
+  label: 'Hours'
 }
+
+export const services = {
+  path: 'json.service_description.service_catagories[0].keys',
```
Comment (sahilshah1) [inline src/constants/properties.js:32]: spelling on `catagories`. also, is it possible that a description has no categories (i.e. incomplete data)? will react handle that gracefully?

**Comment by sahilshah1** [comment]
👍

File: src/constants/properties.js (hunk 1 above)
Comment (vbud) [inline src/constants/properties.js:32]: yeah catagories is spelled wrong in the data unfortunately :(. We will fix when we take a pass at cleaning up and simplifying the data structure.

Comment (vbud) [inline src/constants/properties.js:32]: And `service-catagories` always seems to be there. This is something that will change as well when we simplify the structure.

**Pull request merged**